python/README.md

```diff
@@ -50,21 +50,39 @@ Installation
 4) Build and run the tests:
 
      $ python setup.py build
-     $ python setup.py google_test
+     $ python setup.py test
 
-     To build the C++ implementation run:
-     $ python setup.py build --cpp_implementation
+     To build, test, and use the C++ implementation, you must first compile
+     libprotobuf.so:
+
+     $ (cd .. && make)
+
+     On OS X:
```

Comment:
It does, but dyld (the OS X linker) links the full path at link-time, so
DYLD_LIBRARY_PATH isn't needed in the simple case described here.
On Apr 10, 2015 4:11 PM, "Joshua Haberman" <notifications@github.com> wrote:

> In python/README.md
> <https://github.com/google/protobuf/pull/276#discussion_r28176356>:
>
> >
> > -     To build the C++ implementation run:
> > -     $ python setup.py build --cpp_implementation
> > +     To build, test, and use the C++ implementation, you must first compile
> > +     libprotobuf.so:
> > +
> > +     $ (cd .. && make)
> > +
> > +     On OS X:
>
> I think DYLD_LIBRARY_PATH on OS X behaves like LD_LIBRARY_PATH on Linux?
> http://stackoverflow.com/questions/3146274/is-it-ok-to-use-dyld-library-path-on-mac-os-x-and-whats-the-dynamic-library-s
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/protobuf/pull/276/files#r28176356>.
>